Project.toml

```
@@ -1,10 +1,11 @@
 name = "POMDPFiles"
 uuid = "9cf5b727-2e06-5671-8c87-8c6b0f729d5d"
 repo = "https://github.com/JuliaPOMDP/POMDPFiles.jl"
-version = "0.2.3"
+version = "0.3.0"
```

```suggestion
version = "0.2.4"
```
I do not believe this is a breaking change (correct me if I'm wrong). Before 1.0, non-breaking changes should be indicated by incrementing the patch version. https://pkgdocs.julialang.org/v1/compatibility/#compat-pre-1.0